### PR TITLE
Add TimTales scraper

### PR DESCRIPTION
**Pull request by Jacintocuenca**

### CODE REVIEW

**Comment by bnkai** [comment]
There is already a PR for TimTales #139 
Given that this includes a performer scraper  though and most xpaths are similar we could switch to this instead.

File: scrapers/TimTales.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          - regex: \s–\s.*$
+            with:
+        parseDate: January 02, 2006
+      Details:
+        selector: $info/p[@class="bodytext"]/text()
+        concat: " "
+      Tags:
+        Name: $info/p[@class="categories" and contains(text(), "Categories:")]/a/text()
+      Performers:
+        Name: $info/p[@class="categories" and contains(text(), "Men:")]/a/text()
```
Comment (bnkai) [inline scrapers/TimTales.yml:32]: for some scenes , performers aren't anchors
eg `https://www.timtales.com/videos/2-for-1/` `https://www.timtales.com/videos/rocco-fucks-alejandro-dumas/`
The below seems to fix that
```yaml
      Performers:
        Name: 
          selector: $info/p[@class="categories" and contains(text(), "Categories:")]
          postProcess:
            - replace:
               - regex: "Men:"
                 with: ""
               - regex: "(,)\\s+"
                 with: "$1"
          split: ","
```

File: scrapers/TimTales.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      Title: $info/h1/text()
+      Date:
+        selector: $info/p[@class="date"]/text()
+        replace:
+          - regex: \s–\s.*$
+            with:
+        parseDate: January 02, 2006
+      Details:
+        selector: $info/p[@class="bodytext"]/text()
+        concat: " "
```
Comment (bnkai) [inline scrapers/TimTales.yml:28]: `concat: "\n\n"` instead maintains line breaks in description

File: scrapers/TimTales.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      Tags:
+        Name: $info/p[@class="categories" and contains(text(), "Categories:")]/a/text()
+      Performers:
+        Name: $info/p[@class="categories" and contains(text(), "Men:")]/a/text()
+      Image:
+        selector: //div[@class="video player no-volume play-button"]/@style
+        replace:
+          - regex: .+'(.+)'.+
+            with: $1
+      Studio:
```
Comment (bnkai) [inline scrapers/TimTales.yml:38]: If the studio is always `Timtales`
we can use this instead
```yaml
      Studio:
        Name:
          fixed: "Timtales"
```

**Comment by Jacintocuenca** [comment]
Oh, sorry I overlooked the existing PRs! What were the odds...

Thanks for the feedback!

**Pull request merged**